Gemfile

```diff
@@ -41,6 +41,7 @@ gem "highline",                       "~>1.6.21",      :require => false
 gem "inifile",                        "~>3.0",         :require => false
 gem "kubeclient",                     "~>2.4.0",       :require => false # For scaling pods at runtime
 gem "linux_admin",                    "~>1.2.0",       :require => false
+gem 'log_decorator'                   "~>0.1",         :require => false
```

Comment:
For consistency can you use double-quotes?

Reply:
and a comma after `log_decorator`

Reply:
@Fryguy ah good point. Again I just took it from the other gem. Consistency everywhere!